kratos/geometries/tetrahedra_3d_4.h

```diff
@@ -758,6 +758,115 @@ template<class TPointType> class Tetrahedra3D4 : public Geometry<TPointType>
       return normFactor * Volume() / std::pow(std::sqrt(1.0/6.0 * (sa + sb + sc + sd + se + sf)), 3.0);
     }
 
+
+    /** Calculates the min dihedral angle quality metric.
+     * Calculates the min dihedral angle quality metric.
+     * The min dihedral angle is min angle between two faces of the element 
+     * In radians
+     * @return [description]
+     */
+    virtual double MinDihedralAngle() const override {
+      Vector dihedral_angles(6);
+      ComputeDihedralAngles(dihedral_angles);
+      double min_dihedral_angle = 1000.0;
+      for (unsigned int i = 0; i < 6; i++)
```

Comment:
```suggestion
      for (unsigned int i = 0; i < 6; ++i)
```

Reply:
why is this style preferred?

Reply:
https://google.github.io/styleguide/cppguide.html#Preincrement_and_Predecrement

doesn't matter in this case but in some cases it matters, so it is a good habbit